src/ResourceManager/TrafficManager/Commands.TrafficManager2/Commands.TrafficManager.csproj

```diff
@@ -111,6 +104,21 @@
       <SubType>Designer</SubType>
     </EmbeddedResource>
   </ItemGroup>
+  <ItemGroup>
+    <Reference Include="Microsoft.Azure.Management.TrafficManager, Version=2.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
+      <HintPath>..\..\..\packages\Microsoft.Azure.Management.TrafficManager.2.4.0-preview\lib\net452\Microsoft.Azure.Management.TrafficManager.dll</HintPath>
+    </Reference>
+    <Reference Include="Microsoft.Rest.ClientRuntime, Version=2.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
```

Comment:
@allencal you can remove the references to `Microsoft.Rest.ClientRuntime`, `Microsoft.Rest.ClientRuntime.Azure`, `Newtonsoft.Json` and `System.Runtime` added in this file